tests/bootstrap.php

```diff
@@ -31,10 +31,5 @@
 }
 
 if (file_exists($file = __DIR__.'/custom_bootstrap.php')) {
-    /**
-     * @psalm-suppress MissingFile,UnusedPsalmSuppress
-     *
-     * @see https://github.com/vimeo/psalm/issues/3886
-     */
```

Comment:
You can't touch this file.

Reply:
my psalm was telling me "unneeded suppression"

Reply:
But this class is manager by dev-kit, if you remove lines here, dev-kit will add them again. Can you please add them to see if the build fails?